entrypoint.sh

```diff
@@ -28,28 +35,34 @@ echo "Setting git variables"
 export GITHUB_TOKEN=$API_TOKEN_GITHUB
 git config --global user.email "$INPUT_USER_EMAIL"
 git config --global user.name "$INPUT_USER_NAME"
+rand=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 4 ;)
 
 echo "Cloning destination git repository"
 git clone "https://$API_TOKEN_GITHUB@github.com/$INPUT_DESTINATION_REPO.git" "$CLONE_DIR"
 
 echo "Copying contents to git repo"
-mkdir -p $CLONE_DIR/$INPUT_DESTINATION_FOLDER/
-cp $INPUT_SOURCE_FOLDER "$CLONE_DIR/$INPUT_DESTINATION_FOLDER/"
+
+
+for i in "${!input_source[@]}"; do
+    mkdir -p $CLONE_DIR/${input_dest[i]}/
+    cp -r ${input_source[i]} "$CLONE_DIR/${input_dest[i]}"
```

Comment:
See https://github.com/careofvitamins/web/pull/9650#pullrequestreview-1353417804

```suggestion
    cp -r ${input_source[i]}/* "$CLONE_DIR/${input_dest[i]}"
```